crates/matrix-sdk/CHANGELOG.md

```diff
@@ -797,24 +792,22 @@ simpler methods:
 
 - Improve documentation of `Client::observe_events`.
 
-
 ### Features
 
-
 - Add `create_room_alias` function.
 
 - `Client::cross_process_store_locks_holder_name` is used everywhere:
- - `StoreConfig::new()` now takes a
-   `cross_process_store_locks_holder_name` argument.
- - `StoreConfig` no longer implements `Default`.
- - `BaseClient::new()` has been removed.
- - `BaseClient::clone_with_in_memory_state_store()` now takes a
-   `cross_process_store_locks_holder_name` argument.
- - `BaseClient` no longer implements `Default`.
- - `EventCacheStoreLock::new()` no longer takes a `key` argument.
- - `BuilderStoreConfig` no longer has
-   `cross_process_store_locks_holder_name` field for `Sqlite` and
-   `IndexedDb`.
+- `StoreConfig::new()` now takes a
+  `cross_process_store_locks_holder_name` argument.
+- `StoreConfig` no longer implements `Default`.
+- `BaseClient::new()` has been removed.
+- `BaseClient::clone_with_in_memory_state_store()` now takes a
+  `cross_process_store_locks_holder_name` argument.
+- `BaseClient` no longer implements `Default`.
+- `EventCacheStoreLock::new()` no longer takes a `key` argument.
+- `BuilderStoreConfig` no longer has
+  `cross_process_store_locks_holder_name` field for `Sqlite` and
+  `IndexedDb`.
```

Comment:
Nop, those are a sublist of the previous item.

Reply:
Please, yes, revert these modifications :-).

Reply:
fixup: [`4d21c7f603898469af4c12be78f1d87ff9ea1d98`](https://github.com/matrix-org/matrix-rust-sdk/pull/5946/commits/4d21c7f603898469af4c12be78f1d87ff9ea1d98)